wagtail/admin/static_src/wagtailadmin/scss/userbar.scss

```
@@ -80,7 +80,7 @@ $positions: (
   width: $size-home-button;
   height: $size-home-button;
   margin: 0;
-  overflow: hidden;
+  //overflow: hidden;
   background-color: $color-white;
```

```suggestion
  background-color: $color-white;
```

Let’s remove this altogether if it does nothing.